src/NewField.cpp

```diff
@@ -277,20 +283,28 @@ void NewFieldColumn::take_over_mods(NewFieldColumn& other)
 #undef CPY
 }
 
-void NewFieldColumn::set_defective_mode()
+void NewFieldColumn::set_defective_mode(bool mode)
```

Comment:
If that is the case, then can the variable be renamed so that it's clear what type of mode it is?